.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,992 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
also delete this inspection profile in your settings, you should only have `kiwixAndroidInspections`

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/35a0bffb478e279758c4096d8ced4ea571e3771e)